assembl/graphql/permissions_helpers.py

```diff
@@ -0,0 +1,27 @@
+from pyramid.security import Everyone
+from pyramid.httpexceptions import HTTPUnauthorized
+from assembl.auth.util import get_permissions
+
+
+def make_cls_permissions_querier(cls, request):
```

Comment:
I agree that a helper is useful but maybe a simple `check_permission(cls, permission, request)` function would have been simpler here. You don't "gain" much since you are just accessing attributes/dict values in the outer function.

Reply:
even more since the produced functions will probably only be called once, will change